TipCatDotNet.Api/Controllers/PaymentController.cs

```diff
@@ -34,10 +48,32 @@ public async Task<IActionResult> Get(string memberCode)
         /// <param name="paymentRequest">Payment request</param>
         /// <returns></returns>
         [HttpPost]
-        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(typeof(PaymentDetailsResponse), StatusCodes.Status200OK)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
         public async Task<IActionResult> Pay([FromBody] PaymentRequest paymentRequest)
-            => NoContentOrBadRequest(await _paymentService.Pay(paymentRequest));
+            => OkOrBadRequest(await _paymentService.Pay(paymentRequest));
+
+
+        /// <summary>
+        /// Capture the payment by id.
+        /// </summary>
+        /// <param name="paymentId">Payment id</param>
+        /// <returns></returns>
+        [HttpPost("{paymentId}/capture")]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> Capture(string paymentId)
+            => NoContentOrBadRequest(await _paymentService.Capture(paymentId));
+
+
+        [HttpPost("webhook")]
```

Comment:
Could be more meaningful

Reply:
We can separate webhook by many endpoints for each event-trigger. And collect them an another controller.

Reply:
Nah, that's not usually pays off

Reply:
These endpoint should count while deploy, so keeping numbers low is reasonable